.gitattributes

```diff
@@ -15,8 +17,7 @@
 # Declare files that will always have LF line endings on checkout.
 *.sh eol=lf
 *.plist eol=lf
-MacOS_users_read_this_first.txt eol=lf
-triplea_maps.xml eol=lf
+*.properties eol=lf
```

Comment:
CRLF endings was giving a lot of trouble for scripting. Doesn't make sense to force "^M" characters to then read it back and deal with stripping it away.